uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/test/java/org/uberfire/ext/wires/core/grids/client/util/CellContextUtilitiesTest.java

```diff
@@ -170,15 +174,52 @@ public void testMakeCellRenderContextOneRow() {
 
         assertThat(context.getAbsoluteCellY())
                 .as("Should be headers height")
-                .isEqualTo(headerRowsHeight);
+                .isEqualTo(HEADER_HEIGHT);
     }
 
     @Test
     public void testMakeCellRenderContextThreeRows() {
-        final double headerRowsHeight = 100.0;
-        final BaseGridRow row1 = new BaseGridRow();
-        final BaseGridRow row2 = new BaseGridRow();
-        final BaseGridRow row3 = new BaseGridRow();
```

Comment:
few more tests create `new BaseGridRow()` as part of test method, do you want to update them to use `row1`, `row2` ...?